crates/ruff/src/rules/pyflakes/snapshots/ruff__rules__pyflakes__tests__F401_F401_6.py.snap

```diff
@@ -20,15 +20,15 @@ F401_6.py:7:25: F401 [*] `.background.BackgroundTasks` imported but unused
 9 8 | # F401 `datastructures.UploadFile` imported but unused
 10 9 | from .datastructures import UploadFile as FileUpload
 
-F401_6.py:10:43: F401 [*] `.datastructures.UploadFile` imported but unused
+F401_6.py:10:43: F401 [*] `pyflakes.datastructures.UploadFile` imported but unused
```

Comment:
I don't know if this is good or bad. We now show the fully resolved module path, rather than the relative path.

Reply:
Hmm. I would prefer keeping the import the same as it is in the source document. It can otherwise be confusing... You search for `pyflakes` and you then end up removing the wrong import.

Reply:
I don't actually have a great way to solve this. I may have to revert to storing relative paths for relative imports, and doing the relative path resolution in `resolve_call_path` as before. Or, when we create the name for the diagnostic, I could format the name based on the statement. The crux of the issue is that we don't store whether the import was relative on the binding right now, and adding that information will increase the size of the struct.

Reply:
Alright, this behavior is restored. I moved relative path resolution back into the semantic model, so we now store relative call paths (like `[".", "foo", "bar"]` for `from .foo import bar`).